pkg/kev/converter/kubernetes/transform_test.go

```diff
@@ -737,21 +737,30 @@ var _ = Describe("Transform", func() {
 
 		When("project service label instructing to expose the k8s service is specified as empty string", func() {
 			BeforeEach(func() {
-				projectService.Labels = composego.Labels{
-					config.LabelServiceExpose: "",
-				}
+				projectService.SvcK8sConfig.Service.Expose.Domain = ""
+				// projectService.Labels = composego.Labels{
```

Comment:
can drop commented out lines here and below I think?

Reply:
Fixed in #525